packages/neo-one-server-plugin-project/src/initCommand.ts

```diff
@@ -7,37 +7,54 @@ import jsonFile from 'json-file-plus';
 import * as path from 'path';
 import { loadProjectConfig } from './utils';
 
+interface InitDataFileOpts {
+  readonly folder: string;
+  readonly name: string;
+  // tslint:disable-next-line:no-any
+  readonly data: any;
```

Comment:
We can maybe make this more strict than `any` since it is either expecting a ProjectConfig https://github.com/neo-one-suite/neo-one/blob/6030f892c6d77419cda83bd0bcb55e896118d0e1/packages/neo-one-server-plugin-project/src/types.ts#L36 or a tsconfig.

Reply:
we could do the ProjectConfig, but TSConfig as a type does not appear to exist, and I'd ask @dicarlo2  if he thinks it's worth while to define the shape of a TSConfig.

Reply:
Just make `data` a string and `JSON.stringify` before passing it into the function.

Reply:
+JSON.stringify(data) 
+JSON.parse(data)